samples/setup_helper.py

```diff
@@ -0,0 +1,63 @@
+####
+# This script provides the shared login logic and server creation for all other samples.
```

Comment:
It's fine either way for me. 
I just thought it would be easier to have centralized login to give support for PersonalAccessTokens vs copy-pasting it into each sample. I saw different samples take different arguments to log in, so I wanted a consistent experience as a cli user trying it out.

I'm not sure about the style followed by the project in general. If you think plain is better, I can change it back.

Reply:
In that case, I would keep this out of the change -- each script can be stand alone and therfore copy/pasted around to the users needs.